### DISCUSSION TITLE
My application is approved, but haven't got the benefits.

### DISCUSSION THREAD

**Discussion by rg991** [community/community#120733] [GitHub Education] [answered]
### Hi everyone,

My 2nd application is approved but I didn't get any mail for Approval or Benefits?
How much time does it take? Cause I've seen people here, getting their requests approved but not getting benefits for a long time.
![Screenshot 2024-04-22 191442](https://github.com/community/community/assets/112370361/ea2c8a2e-bc59-41b0-925a-2d95b3be8d65)

**Comment by eddysantiagoo** [comment, marked as answer]
Hi @rg991, i had the same issue when i made my verification of github education, i did it by adding my institutional email and a certificate issued by my university indicating that I was studying at that university, but it took me around 1 week to activate the benefits

If the answer helps you please mark this answer as the correct answer

**Reply by rg991** [reply]
You mean, your request got approved.
After a week of approval then you got the benefits ?

**Reply by eddysantiagoo** [reply]
@rg991 Yes my request was approved but i did not get the benefits immediately, it can take a week or even 2 weeks for the benefits to be activated, the way it works is curious because github can approve your verification immediately but take one or 2 weeks to activate the benefits haha

**Reply by eddysantiagoo** [reply]
After that, you'll receive all benefits and email confirmation

**Reply by rg991** [reply]
Thank for replying, yeah I'll wait too.
Btw, How many requests did you make? (Mine got approved on 2nd request)

**Reply by eddysantiagoo** [reply]
@rg991 I get the approved in the first request haha, hey if the answer help you don't forget to mark this answer as the correct answer, there are many questions about this same topic

**Reply by rg991** [reply]
Yeah sure, My Only option is to wait now haha

**Reply by eddysantiagoo** [reply]
Sure just be patient @rg991 i am making a challenge of resolve the greatest amount of discussions so if the answer helps you please mark this answer as the correct answer :)

**Reply by rg991** [reply]
Finally got it, Thanks for helping me out :)

**Comment by rg991** [comment]
> Sure just be patient @rg991 i am making a challenge of resolve the greatest amount of discussions so if the answer helps you please mark this answer as the correct answer :)
